### ISSUE TITLE
Minor bug in pprint when using Latex

### ISSUE THREAD

**Issue by MoritzImendoerffer** [sympy/sympy#9313]
```
v_msc= sp.MatrixSymbol('vdot_msc',1,3)
V_msc = sp.diag(*v_msc )
sp.pprint(V_msc)
```

Output:

```
⎡v̇_msc_00      0          0    ⎤
⎢                               ⎥
⎢    0      v̇_msc_01      0    ⎥
⎢                               ⎥
⎣    0          0      v̇_msc_02⎦
```

System: Ipython Jupiter 3.0.0-f75fda4
Python 2.7.9 |Anaconda 2.1.0 (64-bit)| (default, Mar  9 2015, 16:20:48) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)]

**Comment by cbm755** [comment]
Probably b/c of the dot.  Similar but less extreme problem happens with `exp(x)` inside matrices.

**Comment by lilithxxx** [comment]
I want to take up this issue. Please guide me how to do it

**Comment by RishobGuhaPatra** [comment]
I want to take up this issue. Please guide me what to do! @cbm755

**Comment by oscargus** [comment]
@sharma-kunal It did not close the issue. A hint is to identify all combining accents first and then do not include those when computing the width. See https://en.wikipedia.org/wiki/Combining_Diacritical_Marks

As they are in a single continuous numerical range it should be quite feasible, although Unicode in itself is a bit challenging sometimes.

**Comment by oscargus** [comment]
Seems to be solved in #12205 although some tests are required to merge it.
